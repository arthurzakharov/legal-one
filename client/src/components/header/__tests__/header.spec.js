import React from 'react';
import {BrowserRouter} from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from '../header.component';

describe('Header: ', () => {
  test('render', () => {
    const component = renderer
      .create(
        <BrowserRouter>
          <Header />
        </BrowserRouter>
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
