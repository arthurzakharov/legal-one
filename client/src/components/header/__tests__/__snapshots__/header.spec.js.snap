// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header:  render 1`] = `
<header
  className="Header"
>
  <ul
    className="HeaderList"
  >
    <li
      className="HeaderListItem"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        General report
      </a>
    </li>
    <li
      className="HeaderListItem"
    >
      <a
        href="/agent"
        onClick={[Function]}
      >
        Agents list
      </a>
    </li>
  </ul>
</header>
`;
